src/main/java/mate/controller/driver/CreateDriverController.java

```diff
@@ -0,0 +1,36 @@
+package mate.controller.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/create")
+public class CreateDriverController extends HttpServlet {
+    private static final Injector injector =
+            Injector.getInstance("mate");
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/drivers/create.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Driver driver = new Driver();
+        String name = req.getParameter("driver_name");
+        String licenseNumber = req.getParameter("license_number");
+        driver.setName(name);
+        driver.setLicenseNumber(licenseNumber);
+        driverService.create(driver);
+    }
```

Comment:
After executing the 'create' method, the user is not redirected to any page. This might cause confusion for the user as no visual feedback will be provided after submitting the form.